labs/outer-loop-aks-azure.md

```diff
@@ -33,6 +37,16 @@ echo $MY_BRANCH
 
     ```
 
+- Validate user role on subscription
+
+  ```bash
+  # make sure your RoleDefinitionName is Contributor or Owner to succssfully create resources in this lab
+  principal_name=$(az account show --query "user.name" --output tsv | sed -r 's/[@]+/_/g')
+
+  az role assignment list --query "[].{principalName:principalName, roleDefinitionName:roleDefinitionName, scope:scope} | [? contains(principalName,'$principal_name')]" -o table
```

Comment:
This will provide user role on given subscription to make sure resource creation doesn't fail

![image](https://user-images.githubusercontent.com/71272427/197246218-5d5363ea-7455-4795-b95d-88baaba6f99d.png)